api/src/main/java/javax/enterprise/concurrent/ContextService.java

```diff
@@ -34,8 +34,8 @@
  * <li>The proxy instance will implement all of the interfaces specified on the
  *     {@code createContextualProxy} methods.
  * <li>The object to have a proxy instance created for should not be a 
- *     component managed by the Java&trade; EE Product Provider, such as a web
- *     component or an EJB.
+ *     component managed by the Jakarta&trade; EE Product Provider, such as a web
+ *     component or a Jakarta&trade; Enterprise Bean.
```

Comment:
I don't think we need the trademark symbol after "Jakarta".

Reply:
In general or just for Jakarta Enterprise Bean? tm makes sense in general as I assume Eclipse Foundation are asserting this is a trademark?

Reply:
You should probably check with Eclipse to see what their trademark guidelines are,
but people often got carried away with the TM and did even more than the Oracle
trademark guidelines required.

Reply:
Check the guidelines first usage should have tm and subsequent uses don't need to. Additional prominent usages do.